plutarch-docs/Typeclasses/PConstant and PLift.md

```diff
@@ -91,7 +91,7 @@ This comes in three flavors:
 - Plutarch type represented **indirectly** by a builtin type that **is not** `Data` (`DefaultUniData`)
   ==> `DerivePConstantViaNewtype`
 
-  *Ex:* `PPubKeyHash` is a newtype to a `PByteString`, and `PByteString` is _directly_ represented as a
+  *Ex:* `PPubKeyHash` is a newtype to a `PDataNewtype PByteString`, and `PByteString` is _directly_ represented as a
```

Comment:
Same thing here, I think using PByteString is no longer too straightforward due to `PDataNewtype`